src/components/Description/DescriptionAndButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from "react";
+import { Button } from "../Button";
+import { Description } from "../Description/Description";
+
+interface IProps {
+  isLoggedIn?: boolean;
+}
+
+export const DescriptionAndButton: FC<IProps> = ({ isLoggedIn = false }) => {
+  const descriptionText = isLoggedIn
+    ? "Pri viacmesačnom predpladtnom zľava až 20%"
+    : "Vyskúšajte si CHAT. Registrujte sa a získajte prvú hodinu zdarma.";
+
+  const buttonText = isLoggedIn ? "Nákup kreditov" : "Registrovať sa";
```

Comment:
we don't need this variable, we can do this in line 19 (where this variable is used)